cvat/apps/engine/migrations/0041_auto_20210827_0258.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('engine', '0040_cloud_storage'),
+        ('engine', '0041_auto_20210813_0853'),
```

Comment:
Not sure if this is the right way, maybe reordering the migration is here. Please make sure this works if migration "0040" has already been applied.